src/main/java/com/avispl/symphony/dal/communicator/biamp/sagevue/SageVueCommunicator.java

```diff
@@ -9,6 +9,7 @@
 import com.avispl.symphony.api.dal.dto.monitor.ExtendedStatistics;
 import com.avispl.symphony.api.dal.dto.monitor.Statistics;
 import com.avispl.symphony.api.dal.dto.monitor.aggregator.AggregatedDevice;
+import com.avispl.symphony.api.dal.error.CommandFailureException;

```

Comment:
Github won't let me comment on things that have not changed, hence commenting here:

1. Please update copyright to 2020-2021 (above)
2. Please add javadoc at class level (below)